game/scripts/vscripts/modifiers/player/enable_player_modifier.lua

```diff
@@ -243,6 +248,12 @@ function EnablePlayerModifier(hEntity)
 		LinkLuaModifier("modifier_player_76", "modifiers/player/modifier_player_76", LUA_MODIFIER_MOTION_NONE)
 		hEntity:AddNewModifier(hEntity, nil, "modifier_player_76", {})
 	end
+	if AsproseSteamAccountID[steamAccountID] then
+		LinkLuaModifier("modifier_player_Asprose_CD","modifiers/player/modifier_player_Asprose_CD",LUA_MODIFIER_MOTION_NONE)
```

Comment:
```suggestion
		LinkLuaModifier("modifier_player_Asprose","modifiers/player/modifier_player_Asprose",LUA_MODIFIER_MOTION_NONE)
```
合并成一个buff后 后面不需要加`_CD`